lavamoat/build-system/policy.json

```diff
@@ -1,12 +1,26 @@
 {
   "resources": {
+    "3box>ipfs>fsm-event>fsm>split": {
```

Comment:
I'm surprised to see 3box here. We don't use it in the build system.

Reply:
Seems like a bug. We do use `split` in the build system, but this path is incorrect.

Reply:
the name used is the shortest dependency path of all dependency paths

Reply:
Hmm.... if this path doesn't represent the actual path used in the bundle, what is its purpose? i.e. how does it improve security.

Reply:
@Gudahtt lavamoat doesn't look at versions specifically, the package manager does the deduplication. lavamoat will load the dependency graph and find the multiple paths pointing at the same location on disk. it will use the shortest name based on the dependency paths for that disk location.

Reply:
I see, so it is in effect listing one entry per package install location. And it uses the shortest dep path for each location as the identifier.